src/App.js

```diff
@@ -101,6 +101,18 @@ const App = () => {
     readApartments()
   }, [])
 
+  useEffect(() => {
+    readApartment()
+  }, [])
```

Comment:
Looks like you have a useEffect already calling readApartments on lines 96 - 102